### PR TITLE
updated swagger schema definitions

### PR DESCRIPTION
**Pull request by Akrish17**
[updating swagger spec for delete category]

#### 📲 What

Updated delete item endpoint description for swagger spec

#### 🤔 Why

Why it's needed, background context.

#### 🛠 How

More in-depth discussion of the change or implementation.

#### 👀 Evidence

Screenshots / external resources / links / etc.
Link to documentation updated with changes impacted in the PR

#### 🕵️ How to test

Notes for QA

#### ✅ Acceptance criteria Checklist

- [ ] Code peer reviewed?
- [ ] Documentation has been updated to reflect the changes?
- [ ] Passing all automated tests, including a successful deployment?
- [ ] Passing any exploratory testing?
- [ ] Rebased/merged with latest changes from development and re-tested?
- [ ] Meeting the Coding Standards?

### CODE REVIEW

File: java/src/main/java/com/xxAMIDOxx/xxSTACKSxx/menu/api/v1/DeleteCategoryController.java [hunk 1]
```diff
@@ -17,12 +17,12 @@
...
 public interface DeleteCategoryController {
 
-  @DeleteMapping(produces = "application/json")
+  @DeleteMapping(produces = "application/json; charset=utf-8")
   @Operation(
       tags = "Category",
-      summary = "Update a category in the menu",
-      description = "Update a category to menu",
-      operationId = "UpdateMenuCategory",
+      summary = "Removes a category and it's items from menu",
```
Comment (peet-j) [inline java/src/main/java/com/xxAMIDOxx/xxSTACKSxx/menu/api/v1/DeleteCategoryController.java:23]: `its`

File: java/src/main/java/com/xxAMIDOxx/xxSTACKSxx/menu/api/v1/CreateCategoryController.java [hunk 2]
```diff
@@ -20,7 +20,7 @@
 @RequestMapping("/v1/menu/{id}/category")
 public interface CreateCategoryController {
 
-  @PostMapping(consumes = "application/json", produces = "application/json")
+  @PostMapping(consumes = "application/json", produces = "application/json; charset=utf-8")
```
Comment (peet-j) [inline java/src/main/java/com/xxAMIDOxx/xxSTACKSxx/menu/api/v1/CreateCategoryController.java:23]: What's the motivation behind all the charset values?

Comment (Akrish17) [reply java/src/main/java/com/xxAMIDOxx/xxSTACKSxx/menu/api/v1/CreateCategoryController.java:23]: the pact test expects "application/json; charset=utf-8".

Comment (sureshkrishnan-mr) [reply java/src/main/java/com/xxAMIDOxx/xxSTACKSxx/menu/api/v1/CreateCategoryController.java:23]: I think it's a pact issue.

UTF-8  is the default one for the rest controller.We don't need specify this in our controller but it looks like the client used by pact uses a ASCII charset if charset is not defined by the producer.

**Review by sureshkrishnan-mr — APPROVED** [review]

**Pull request merged**